### ISSUE TITLE
meteor 1.x support

### ISSUE THREAD

**Issue by avalanche1** [bminer/meteor-blade#9]
Hello!
Do you intend to update blade for 1.x? We very much need an html preprocessor...)

**Comment by bminer** [comment]
Yes.  I have intentions to drastically re-vamp Blade and release version 4.0 this year.  I'll post more information on this project's README.md.  Please feel free to discuss any features that you want to see with me directly, on the wiki, or in the issue tracker.
